src/mongocxx/collection.cpp

```diff
@@ -426,6 +430,8 @@ stdx::optional<bsoncxx::document::value> collection::find_one_and_replace(
         flags |= ::MONGOC_FIND_AND_MODIFY_RETURN_NEW;
     }
 
+    // TODO: use options.max_time() when available in the C driver
```

Comment:
Sorry for the jargon. Release note. I'd like to add this as a "known issue". We don't really have a plan for how to do that, yet.